main.js

```diff
@@ -0,0 +1,28 @@
+function verifyEmail() {
+  var status= document.getElementById('emailStatus');
+
+  var emailPattern=/^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\.([a-zA-Z])+([a-zA-Z])+/;
+  if (emailPattern.test(subscribeField.value)) {
+    status.classList.remove("notApproved");
```

Comment:
Does this work as an argument? I would think you would need to use `elEmail.value` here or the full `document.get...`